pom.xml

```diff
@@ -157,11 +158,10 @@
         <ddf-spatial>DDF Spatial</ddf-spatial>
         <ddf-ui>DDF Search UI</ddf-ui>
 
-        <asciidoctor.maven.plugin.version>1.5.3</asciidoctor.maven.plugin.version>
-        <asciidoctorj.version>1.5.4.1</asciidoctorj.version>
-        <asciidoctorj.diagram.version>1.5.0</asciidoctorj.diagram.version>
-        <asciidoctorj.pdf.version>1.5.0-alpha.11</asciidoctorj.pdf.version>
-        <jruby.version>1.7.25</jruby.version>
+        <asciidoctor.maven.plugin.version>1.5.2</asciidoctor.maven.plugin.version>
```

Comment:
This should not be changed. You're rolling back a patch that fixed builds in Windows.